src/devtools/views/Components/Element.js

```diff
@@ -66,6 +69,21 @@ export default function ElementView({ index, style }: Props) {
     [id, selectElementByID]
   );
 
+  const rendererID = store.getRendererIDForElement(element.id) || null;
```

Comment:
nit `store.getRendererIDForElement` already returns `null` if it can't find a match

Reply:
Copy pasted. Want me to change it in both places?

Reply:
Ah,  gotcha.  Yeah if you don't mind 😄